versioned_docs/version-22.04/getting-started/create-graphical-view.md

```diff
@@ -89,22 +86,22 @@ If your IT is geographically spread over a city, country or even the world, you
 may want to visualize it in the most appropriate way to get insights on your IT
 infrastruture status.
 
-This can be done easily by using the "GeoView", the only pre-requisites is to
-have define Geo-coordinates paramaters on your hosts (in the Extended
+This can be done easily by using the **GeoView**. The only prerequisites is to
+have defined Geo-coordinates parameters on your hosts (in the Extended
 information).
```

Comment:
```suggestion
This can be done easily by using the **GeoView**. The only prerequisite is to
have defined Geo-coordinates parameters on your hosts (which can be consulted
and modified in the "Host Extended Infos" tab).
```